torch/csrc/utils.cpp

```diff
@@ -551,3 +551,33 @@ void THPPointer<THPGenerator>::free() {
 }
 
 template class THPPointer<THPGenerator>;
+
+static bool backCompatBroadcastWarn = false;
```

Comment:
Someone better at C++ than I can evaluate whether or not this works as a global variable?